nvflare/app_common/filters/params_converter_filter.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Union
+
+from nvflare.apis.dxo_filter import DXO, DXOFilter
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import Shareable
+from nvflare.app_common.abstract.params_converter import ParamsConverter
+
+
+def _get_params_converter(params_converter_ids: List[str], fl_ctx: FLContext) -> List[ParamsConverter]:
+    filters = []
```

Comment:
These should be "converters", not "filters"

Reply:
fixed.